src/commands/news.ts

```diff
@@ -25,20 +25,17 @@ export default class News {
   async run(interaction: ChatInputCommandInteraction) {
     const db = this.db;
     const newsletterTable = await getNewsTable(db);
-
-    const nebulaId = "903852579837059113"
     let page = interaction.options.getNumber("page") ?? 1;
 
-    const news = await newsletterTable.get(`${nebulaId}.news`).then(
-      (news: any) => news as any[] ?? []
-    ).catch(() => []);
-    const newsSorted = (Object.values(news) as any[])?.sort((a, b) => b.createdAt - a.createdAt);
+    const news = await newsletterTable
+      .get(`903852579837059113.news`)
```

Comment:
```suggestion
      .get(`903852579837059113.news`) // News of the Nebula server
```
Might be unclear, but I agree using a variable here is not the solution

Reply:
wym

Reply:
Well, it might be unclear that this is the news of nebula server (when we start mp development) but for now it's fine